api/pkg/s3/service.go

```diff
@@ -167,20 +166,7 @@ func HandleS3Error(response *restful.Response, request *restful.Request, err err
 
 func (s *APIService) GetBackendIdFromTier(ctx context.Context, tierName string) string {
```

Comment:
I think this function not required

Reply:
This is resolved and corrected code is pushed..Please review